engine/src/main/java/org/terasology/engine/rendering/dag/RenderGraph.java

```diff
@@ -275,8 +277,8 @@ private void connectFbo(Node toNode, int inputFboId, DependencyConnection fromCo
 
         } else { // if adding new input failed, it already existed - check for connections
             //TODO update
-            logger.info("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..",
-                    toNode.getUri(), inputFboId, fromConnection.getName());
+            logger.atInfo().addArgument(() -> toNode.getUri()).addArgument(inputFboId).addArgument(() -> fromConnection.getName()).
+                    log("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..");
```

Comment:
simple getter methods
```suggestion
            logger.info("{}.connectFbo({}, {}): Connection already existed. Testing for its connections..",
                    toNode.getUri(), inputFboId, fromConnection.getName()); //NOPMD
```